### ISSUE TITLE
Add visual effect previews

### ISSUE THREAD

**Issue by dhsavell** [beat-machine/beat-webapp#17]
Some of the more complicated effects are getting a bit wordy (i.e. new cut effect), so having visual previews with boxes would probably be useful.

**Comment by JustACuteGirlLookingForAManToFixHerBugs** [comment]
Not sure if you mean the static visual cues or dynamic preview of the end result? It's really hard to understand what happens when you apply two cut effect for example, so visually rendering the end result would be very useful
